guides/source/active_support_instrumentation.md

```diff
@@ -226,6 +226,20 @@ Action View
 }
 ```
 
+### render_collection.action_view
+
+| Key           | Value                 |
+| ------------- | --------------------- |
+| `:identifier` | Full path to template |
+| `:count`      | Size of collection    |
```

Comment:
There's also an optional `:cache_hits` key for the collections rendered with `cached: true`: https://github.com/rails/rails/blob/1de3041e3be7be8477ee28c7ece616a5975321bd/actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb#L17